### PR TITLE
Added fonts to src/fonts and to stylesheet

### PR DESCRIPTION
**Pull request by ghost**
## What does it do?
We have 'Merriweather' and 'SourceSansPro' fonts in src/fonts and can use them on the page.

## Why the change?
For the sake of beauty and readability.

## How can this be tested?
<How is the best way to test this?>

## Where to start code review?
cpp-tour/webpack.config.js:46

## Relevant tickets?
 [Issue #25](https://github.com/Leandros/cpp-tour/issues/25)

## Questions?
<Ask us anything!>

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -43,6 +43,11 @@ const img_loader = {
     },
 }
 
+const text_loader = {
+    test: /\.(eot|svg|ttf|woff|woff2)$/,
```
Comment (DoumanAsh) [inline webpack.config.js]: It is  gonna conflict with `img_loader` since it also test for svg files

File: webpack.config.js [hunk 2]
```diff
@@ -43,6 +43,11 @@ const img_loader = {
     },
 }
 
+const text_loader = {
```
Comment (DoumanAsh) [inline webpack.config.js]: Let's name it font_loader so that it would used only for fonts

**Review by DoumanAsh — CHANGES_REQUESTED** [review]

File: webpack.config.js [hunk 3]
```diff
@@ -43,8 +43,8 @@ const img_loader = {
     },
 }
 
-const text_loader = {
-    test: /\.(eot|svg|ttf|woff|woff2)$/,
+const font_loader = {
+    test: /\.(eot|ttf|woff|woff2)$/,
     loader: 'file-loader?name=public/fonts/[name].[ext]'
```
Comment (DoumanAsh) [inline webpack.config.js]: Use `outputPath` to specify folder(it should be relatieve to our public directory) where to put fonts and leave name as `[name].[ext]`

**Review by DoumanAsh — CHANGES_REQUESTED** [review]
Sorry, I forgot one minor thing.
Please correct it and squash your commits

File: src/styles/common.sss [hunk 4]
```diff
@@ -9,7 +9,7 @@ html
 
 body
     font-size: 20px
-    font-family: Times New Roman, Times, serif
+    font-family: SourceSansPro, Times, serif
```
Comment (Leandros) [inline src/styles/common.sss]: This change will result in a merge conflict, the current `dev` branch also has changed the font family.

**Review by DoumanAsh — APPROVED** [review]

**Comment by Leandros** [comment]
Merged!

**Pull request closed without merging**